src/components/Navbar/Navbar.js

```diff
@@ -1,13 +1,29 @@
 import React from 'react';
 import './Navbar.css';
-import {  NavLink } from 'react-router-dom';
+import {  NavLink, useNavigate } from 'react-router-dom';
+import { useNote } from '../../context/note-context';
+import { useToastContext } from '../../context/toastContext';
+
 
 export default function Navbar() {
+  const { isLogin, setIsLogin , setPinNotes } = useNote();
+  const navigate = useNavigate();
+  const notify = useToastContext();
+
+  const logoutHandler = () => {
+     setIsLogin(false);
+     notify('You are Successfully Logout!' , {type:'success'});
+     setPinNotes([]);
+     navigate('/');
```

Comment:
suggestion: navigate the user to the logout page after clicking the logout button